index.bs

```diff
@@ -599,28 +543,28 @@ If there is more than one [=shape tree location=], they all apply to the same
     </thead>
     <tbody>
       <tr>
-        <td><code class="vocab">st:hasShapeTree</code></td>
+        <td><code class="vocab">st:assigns</code></td>
         <td>Identifies the [=shape tree=] to be associated with 
         the [=managed resource=]</td>
       </tr>
       <tr>
-        <td><code class="vocab">st:hasManagedResource</code></td>
-        <td>Identifies the [=managed resource=] associated with the [=shape tree location=]</td>
+        <td><code class="vocab">st:manages</code></td>
+        <td>Identifies the [=managed resource=] associated with the [=shape tree assignment=]</td>
       </tr>
       <tr>
-        <td><code class="vocab">st:hasRootShapeTreeLocation</code></td>
-        <td>Identifies the [=root shape tree location=]</td>
+        <td><code class="vocab">st:hasRootAssignment</code></td>
```

Comment:
Note that https://github.com/xformativ/shapetrees-java/pull/108#discussion_r742755476 proposes `rootOfInstance`